# SPLC Be Connected Analysis

*Wed 2nd March 2022*

An analysis comparing SPLC's database of people and Capture IT figures.

### Filenames to use

| Data | Filename |
|:------|:----------|
| Capture IT | captureit.csv |
| SPLC Term 1 2022 | splc-T1-2022.csv |
| SPLC 2017 - 2022 | splc-2017-2022.csv |

## Analysis A: Term 1 2022 SPLC Members vs. Capture IT as of 28.02.2020

Import the files into dataframes:

In [1]:
import pandas as pd
dfcm = pd.read_csv('splc-T1-2022.csv')
dfci = pd.read_csv('captureit.csv')

Confirming dataframes shows 537 Term 1 2022 SPLC members and 288 Capture IT learners:

In [2]:
dfcm

Pers_num    Mem_Num Title       Given    Surname                 Street  \
0        3325  202107011   NaN    Caroline      Abbot           1-15 Forward   
1        3186  202103001    Mr       Brian      Adams           1/9 Ridge St   
2        3043  202010016   NaN     Maryon    Allbrook   48b Griffin Crescent   
3        2738  201910006   Mrs      Maryon   Allbrook   48b Griffin Crescent   
4        2747  201910015   NaN         Ian    Allison         2/34 Norton St   
..        ...        ...   ...         ...        ...                    ...   
532      1852  201707200    Ms      Lesley      Young  9/166 Mill Point Road   
533      3461  202202007    Mr     Adrian   Yurisich           5 Otter Court   
534      3462  202202008   Mrs   Virginia   Yurisich           5 Otter Court   
535      2092  201802019   Mrs  Keryn anne       Zeeb    15/63 Mill Point Rd   
536      3375  202109015   NaN        Anne      Zevis    3/11 King Edward St   

          Suburb State   PostC                       Email          Phone  \
0        Manning    WA  6152.0         clucky666@gmail.com            NaN   
1    South perth    WA  6151.0  brianjohnadams50@gmail.com     0400039718   
2        Manning    WA  6152.0    brookwood@westnet.com.au     0406550343   
3        Manning    WA  6152.0    brookwood@westnet.com.au            NaN   
4    South Perth    WA  6151.0  alisonandian01@hotmail.com            NaN   
..           ...   ...     ...                         ...            ...   
532  South Perth    WA  6151.0    lesleyyoung1@bigpond.com            NaN   
533   Willetton     WA  6155.0   adrian.yurisich@gmail.com     0431657566   
534   Willetton     WA  6155.0            vjyuri@gmail.com  0434 113 790    
535  South perth    WA  6151.0        keryn.zeeb@gmail.com     0409089490   
536  South Perth    WA  6151.0         zevises@hotmail.com            NaN   

             Mobile                                             Status  \
0       0408903550.  Member <font class=fontColorHigh>19-Jul-2021 t...   
1                 .  Member <font class=fontColorHigh>1-Jan-2022 to...   
2       0406559342.  Member <font class=fontColorHigh>17-Oct-2021 t...   
3       0452445233.  Member <font class=fontColorHigh>20-Jan-2022 t...   
4       0405327373.  Member <font class=fontColorHigh>12-Apr-2021 t...   
..              ...                                                ...   
532     0403351265.  Member <font class=fontColorHigh>19-Apr-2021 t...   
533     0431657566.  Member <font class=fontColorHigh>5-Feb-2022 to...   
534  0434 113 790 .  Member <font class=fontColorHigh>5-Feb-2022 to...   
535     0409089490.  Member <font class=fontColorHigh>1-May-2021 to...   
536     0423114929.  Member <font class=fontColorHigh>21-Sep-2021 t...   

    Mailouts                  Comments  
0        NaN                       NaN  
1        NaN  Self added by auto enrol  
2        NaN  Self added by auto enrol  
3        NaN  Self added by auto enrol  
4    Email ,                       NaN  
..       ...                       ...  
532  Email ,                       NaN  
533      NaN  Self added by auto enrol  
534      NaN  Self added by auto enrol  
535      NaN  Self added by auto enrol  
536      NaN                       NaN  

[537 rows x 15 columns]

In [3]:
dfci

First Name       Last Name
0          Ada         Channey
1        Adele        Gianotti
2      Adriana         Matusan
3         Alan           Naber
4         Alex        Oudejans
..         ...             ...
283      Wendy          Dawson
284      Wendy         Howlett
285      Wendy            Webb
286    Windsor  Foulkes-Taylor
287        Zoe          Howard

[288 rows x 2 columns]

Make the field names consistent, "First Name" vs. "Given", "Last Name" vs. "Surname":

In [4]:
dfci = dfci.rename(columns={'First Name':'Given','Last Name':'Surname'})

Confirm field names are now consistent:

In [5]:
dfci.head()

Given   Surname
0      Ada   Channey
1    Adele  Gianotti
2  Adriana   Matusan
3     Alan     Naber
4     Alex  Oudejans

In [6]:
dfcm.head()

Pers_num    Mem_Num Title     Given   Surname                Street  \
0      3325  202107011   NaN  Caroline     Abbot          1-15 Forward   
1      3186  202103001    Mr     Brian     Adams          1/9 Ridge St   
2      3043  202010016   NaN   Maryon   Allbrook  48b Griffin Crescent   
3      2738  201910006   Mrs    Maryon  Allbrook  48b Griffin Crescent   
4      2747  201910015   NaN       Ian   Allison        2/34 Norton St   

        Suburb State   PostC                       Email       Phone  \
0      Manning    WA  6152.0         clucky666@gmail.com         NaN   
1  South perth    WA  6151.0  brianjohnadams50@gmail.com  0400039718   
2      Manning    WA  6152.0    brookwood@westnet.com.au  0406550343   
3      Manning    WA  6152.0    brookwood@westnet.com.au         NaN   
4  South Perth    WA  6151.0  alisonandian01@hotmail.com         NaN   

        Mobile                                             Status Mailouts  \
0  0408903550.  Member <font class=fontColorHigh>19-Jul-2021 t...      NaN   
1            .  Member <font class=fontColorHigh>1-Jan-2022 to...      NaN   
2  0406559342.  Member <font class=fontColorHigh>17-Oct-2021 t...      NaN   
3  0452445233.  Member <font class=fontColorHigh>20-Jan-2022 t...      NaN   
4  0405327373.  Member <font class=fontColorHigh>12-Apr-2021 t...  Email ,   

                   Comments  
0                       NaN  
1  Self added by auto enrol  
2  Self added by auto enrol  
3  Self added by auto enrol  
4                       NaN

Find the overlap between Term 2 2022 members and Capture IT learners:

In [7]:
dfovrlp = pd.merge(dfci,dfcm,how='inner',on=['Given','Surname'])

In [8]:
dfovrlp

Given      Surname  Pers_num    Mem_Num Title  \
0       Alex     Oudejans      3340  202108001    Ms   
1     Amanda   Mincherton      1211  201709001    Ms   
2     Amanda      Stewart      1622  202002022    Ms   
3     Andrea      Henning       773  201707088    Ms   
4        Ann     Robinson      1462  201707159    Ms   
..       ...          ...       ...        ...   ...   
89     Trish     Burgoyne      2846  202002027    Ms   
90  Veronica  Shepherdson      1545  201709074    Ms   
91    Warren      McCamey      1144  201712005    Mr   
92     Wendy      Howlett       831  201707097    Ms   
93       Zoe       Howard       826  201801007    Ms   

                           Street         Suburb State   PostC  \
0   14 Swan St  South Perth  6151    South perth   ACT  6151.0   
1                  18 Taylor Road       Nedlands    WA  6009.0   
2                      8 Allen St    South Perth    WA  6151.0   
3            U9 6 Manning Terrace    South Perth    WA  6151.0   
4                  22 Park Street           Como    WA  6152.0   
..                            ...            ...   ...     ...   
89              19 Kittyhawk View       Maylands    WA  6051.0   
90             1/34 Manchester St  Victoria Park    WA  6100.0   
91                   23 Garden St    South Perth    WA  6151.0   
92          3/287 Mill Point Road    South Perth    WA  6151.0   
93                   31 Duncan St  Victoria park    WA  6100.0   

                           Email         Phone         Mobile  \
0         alexoudejans@gmail.com    0432663018    0432663018.   
1        amincher@bigpond.net.au    0450909747    0450909747.   
2    amanda.jayne@hotmail.com.au    0401203024    0401203024.   
3        henning8@bigpond.net.au  0428 311 074  0428 311 074.   
4   ann.samphier@optusnet.com.au      94502445    0403803220.   
..                           ...           ...            ...   
89      trish.burgoyne@gmail.com    0415511477    0415511477.   
90     vshepherdsonnnh@gmail.com     9361 0678    0413792749.   
91       lynwar23@bigpond.net.au           NaN    0418942929.   
92   wendyhowlett@westnet.com.au     9368 1018              .   
93      cellerco@optusnet.com.au           NaN    0404936312.   

                                               Status Mailouts  \
0   Member <font class=fontColorHigh>5-Aug-2021 to...      NaN   
1   Member <font class=fontColorHigh>1-Jan-2022 to...      NaN   
2   Member <font class=fontColorHigh>15-Mar-2021 t...      NaN   
3   Member <font class=fontColorHigh>31-Dec-2017 t...      NaN   
4   Member <font class=fontColorHigh>9-Mar-2021 to...      NaN   
..                                                ...      ...   
89  Member <font class=fontColorHigh>4-Nov-2021 to...      NaN   
90  Member <font class=fontColorHigh>5-Jul-2021 to...      NaN   
91  Member <font class=fontColorHigh>20-Jul-2021 t...      NaN   
92  Member <font class=fontColorHigh>10-Jan-2022 t...      NaN   
93  Member <font class=fontColorHigh>12-Jul-2021 t...      NaN   

                                             Comments  
0                            Self added by auto enrol  
1                                                 NaN  
2                                                 NaN  
3                                                 NaN  
4                                                 NaN  
..                                                ...  
89                           Self added by auto enrol  
90  Any messages, please Text Thank you NOT ON EMAIL   
91                                                NaN  
92  <font class=fontColorHot>Credit Expires <input...  
93                                                NaN  

[94 rows x 15 columns]

Result: **94 learners found**. I.e. out of 537 Term 1 2022 members, 94 are learners on Capture IT.

## Analysis B: Full SPLC People Database 2017 - 2022 vs. Capture IT as of 28.02.2020

Import the database of all SPLC records from 2017 - 2022:

In [9]:
dfcm = pd.read_csv('splc-2017-2022.csv')

Confirming dataframe shows 3000+ people in the database, 2017 - 2022:

In [10]:
dfcm

Pers_num  Mem_Num Title     Given   Surname                 Street  \
0         2914      NaN   NaN     Annie       Kim                    NaN   
1         2896      NaN    Mr       Bob   MacMath                    NaN   
2         3325      NaN   NaN  Caroline     Abbot           1-15 Forward   
3            7      NaN    Ms      Anne    Abbott     9 Canning Beach Rd   
4            8      NaN    Ms    Barbie    Abbott     109/16 Freedman Rd   
...        ...      ...   ...       ...       ...                    ...   
3314      1865      NaN   NaN      Peta    Zeller  12 Foreshore Entrance   
3315      3375      NaN   NaN      Anne     Zevis    3/11 King Edward St   
3316      3027      NaN   Mrs     Chloe     Zhang         99 Douglas Ave   
3317      1864      NaN    Ms      Anne      Zion                    NaN   
3318      2689      NaN    Mr    Robert    Zordan   1/52 Mill Point Road   

           Suburb State PostC                      Email     Phone  \
0             NaN   NaN   NaN       kymannie77@gmail.com       NaN   
1     South Perth    WA  6151  robertmacmath@hotmail.com       NaN   
2         Manning    WA  6152        clucky666@gmail.com       NaN   
3      Applecross    WA  6153        anne@abbotts.net.au       NaN   
4          Menora    WA  6050    bjzusman@westnet.com.au  93700235   
...           ...   ...   ...                        ...       ...   
3314       Wilson    WA  6107         petaz@iinet.net.au       NaN   
3315  South Perth    WA  6151        zevises@hotmail.com       NaN   
3316  South perth    WA  6151          flomo96@gmail.com       NaN   
3317       Yokine    WA  6060                        NaN       NaN   
3318  South Perth    WA  6151       rzordan@iinet.net.au       NaN   

             Mobile  Status Mailouts                  Comments  
0     0433 019 781.     NaN      NaN                       NaN  
1       0438326969.     NaN      NaN                       NaN  
2       0408903550.     NaN      NaN                       NaN  
3       0419364928.     NaN      NaN                       NaN  
4                 .     NaN      NaN                       NaN  
...             ...     ...      ...                       ...  
3314    0402985506.     NaN      NaN                       NaN  
3315    0423114929.     NaN      NaN                       NaN  
3316    0481069887.     NaN      NaN  Self added by auto enrol  
3317              .     NaN      NaN                       NaN  
3318    0439399385.     NaN  Email ,                       NaN  

[3319 rows x 15 columns]

Find the overlap between Term 2 2022 members and Capture IT learners:

In [11]:
dfovrlp = pd.merge(dfci,dfcm,how='inner',on=['Given','Surname'])

In [12]:
dfovrlp

Given   Surname  Pers_num  Mem_Num Title  \
0    Adriana   Matusan      2130      NaN    Ms   
1       Alan     Naber      1257      NaN    Mr   
2       Alex  Oudejans      3340      NaN    Ms   
3     Alicia    Zablah      1857      NaN    Ms   
4     Alison     Casey       293      NaN    Ms   
..       ...       ...       ...      ...   ...   
186   Warren   McCamey      1144      NaN    Mr   
187    Wendy    Dawson       446      NaN    Ms   
188    Wendy   Howlett       831      NaN    Ms   
189    Wendy      Webb      3362      NaN   Mrs   
190      Zoe    Howard       826      NaN    Ms   

                            Street         Suburb State PostC  \
0                5 Cloister Avenue           Como    WA  6152   
1                3/34 Mary Street            Como    WA  6152   
2    14 Swan St  South Perth  6151    South perth   ACT  6151   
3               16/9 Clydesdale St           Como    WA  6152   
4                              NaN            NaN    WA   NaN   
..                             ...            ...   ...   ...   
186                   23 Garden St    South Perth    WA  6151   
187                  6A Thurles Ct      Waterford    WA  6152   
188          3/287 Mill Point Road    South Perth    WA  6151   
189                48a Tain Street     Applecross    WA  6153   
190                   31 Duncan St  Victoria park    WA  6100   

                           Email       Phone         Mobile  Status Mailouts  \
0    jadrankamatusan@hotmail.com  0405691410              .     NaN      NaN   
1            anaber@iinet.net.au         NaN  0415 873 862.     NaN      NaN   
2         alexoudejans@gmail.com  0432663018    0432663018.     NaN      NaN   
3        aliciazablah@tpg.com.au      000000    0451728732.     NaN      NaN   
4       alisonfaycasey@gmail.com         NaN              .     NaN      NaN   
..                           ...         ...            ...     ...      ...   
186      lynwar23@bigpond.net.au         NaN    0418942929.     NaN      NaN   
187            dawswen@gmail.com    94505252    0415993361.     NaN      NaN   
188  wendyhowlett@westnet.com.au   9368 1018              .     NaN      NaN   
189          wwebb88@hotmail.com         NaN    0439105735.     NaN      NaN   
190     cellerco@optusnet.com.au         NaN    0404936312.     NaN      NaN   

                                              Comments  
0                                                  NaN  
1                                                  NaN  
2                             Self added by auto enrol  
3                                                  NaN  
4                                                  NaN  
..                                                 ...  
186                                                NaN  
187                                                NaN  
188  <font class=fontColorHot>Credit Expires <input...  
189                           Self added by auto enrol  
190                                                NaN  

[191 rows x 15 columns]

Result: **191 learners found**. I.e. out of 3000+ people in SPLC's database, 191 are learners on Capture IT.